src/app/helpers/page/pageInfo.ts

```diff
@@ -28,6 +27,12 @@ export interface IPageInfo extends Data {
   fullscreen?: boolean;
   resolvers?: ResolverList;
   menus?: Menus;
+
+  /**
+   * Describes where the page will be rendered.
+   * @default RenderMode.Server
```

Comment:
add a comment about how our linkToRoute(?) function plumbs this into the route data for the page